Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/DataStream/Cancellation/CancelProgressFlowNode.cs

```diff
@@ -81,16 +86,13 @@ private JobHandle ScheduleCheckCancelProgressJob(JobHandle dependsOn)
                 m_CancelCompleteData.PendingWriter,
                 m_CancelCompleteDataTargetID,
                 m_ProgressLookupData.DataOwner.WorldUniqueID,
-                m_Parent != null ? m_ParentProgressLookupData.Lookup : default);
+                m_ParentProgressLookupData?.Lookup ?? default);
```

Comment:
I think there's a performance implication to this because it creates an intermediate `Nullable<UnsafeParallelHashMap<EntityProxyInstanceID, bool>>` before hitting the `??`

Reply:
Interesting, I went down a rabbit hole and it appears the `??` is better. It's also what Rider often suggests. 

https://stackoverflow.com/questions/13385503/which-works-faster-null-coalesce-ternary-or-if-statement

Reply:
Huh, ok the compiler is smarter than I thought!

Verified:
https://sharplab.io/#v2:D4AQTAjAsAUCDMACciDCiDetE8d3CiAzgC4BOArgMYmIBqAhgDYUCmAyudSfjljLkF4BuAL69hgwigBKrAGaomDIkQn8hBJIxYcuNRAFkAnp0o0A3BM05rNwnMXLVACgCUdzRps+TZ7ogAvIgAJgoMFEwkViI+OOKxiAk+EhIADmQAlgBuDCSsiI5KKkSIALYA+ibFqjGCEtIALEbumJ6SNrlk5cY6QeVVxjVEAPwAdH76tCMjoeGR0e0AKqykLmW9zG51YqmJTYgrazps/gb5pB6J3poSCaJAA=